src/scripts/install.sh

```diff
@@ -11,11 +15,20 @@ Install() {
 
     BINPATH="${HOME}/.dctlenv/bin"
     if [ ! -d "${HOME}/.dctlenv" ]; then
-      git clone --depth 1 --branch v0.1.8 https://github.com/wbeuil/dctlenv ~/.dctlenv
+      git clone --depth 1 --branch v0.1.9 https://github.com/wbeuil/dctlenv ~/.dctlenv
     fi
 
     VERSION=$(GetVersion)
-    gpg --import driftctl_pubkey.pem
```

Comment:
I'd be in flavor of adding another key file here to avoid relying on keyservers that are not super reliable in general.
Let's rename the old one `driftctl_legacy_pubkey.pem` and add the new one.